pkg/operator/configobserver/node/observe_minimum_kubelet_version_test.go

```diff
@@ -0,0 +1,80 @@
+package node
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	configv1 "github.com/openshift/api/config/v1"
+	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
+	"github.com/openshift/library-go/pkg/operator/events"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	clocktesting "k8s.io/utils/clock/testing"
+)
+
+func TestObserveKubeletMinimumVersion(t *testing.T) {
+	type Test struct {
+		name                   string
+		existingConfig         map[string]interface{}
+		expectedObservedConfig map[string]interface{}
+		minimumKubeletVersion  string
+	}
+	tests := []Test{
+		{
+			name:                   "empty minimumKubeletVersion",
+			expectedObservedConfig: map[string]interface{}{},
+			minimumKubeletVersion:  "",
+		},
+		{
+			name: "set minimumKubeletVersion",
+			expectedObservedConfig: map[string]interface{}{
+				"minimumKubeletVersion": string("1.30.0"),
+			},
+			minimumKubeletVersion: "1.30.0",
```

Comment:
should this value also be prefixed with `v`?

Reply:
actually the min version API doesn't accept a v. only on the kubelet's side do we strip it